docs/project/dogfooding.md

```diff
@@ -155,17 +152,50 @@ $ dotnet publish
+| **OpenSUSE 42 (x64)** | [![][OpenSUSE-42-badge-6.0.X-coreclr]][OpenSUSE-42-version-6.0.X-coreclr]<br>[Runtime-Deps][OpenSUSE-42-runtime-deps-6.0.X-coreclr] ([Checksum][OpenSUSE-42-runtime-deps-checksum-6.0.X-coreclr])<br>[Host][OpenSUSE-42-host-6.0.X-coreclr] ([Checksum][OpenSUSE-42-host-checksum-6.0.X-coreclr])<br>[App Hosts][OpenSUSE-42-apphost-pack-6.0.X-coreclr] ([Checksum][OpenSUSE-42-apphost-pack-checksum-6.0.X-coreclr])<br>[Host FX Resolver][OpenSUSE-42-hostfxr-6.0.X-coreclr] ([Checksum][OpenSUSE-42-hostfxr-checksum-6.0.X-coreclr])<br>[Targeting Pack][OpenSUSE-42-targeting-pack-6.0.X-coreclr] ([Checksum][OpenSUSE-42-targeting-pack-checksum-6.0.X-coreclr])<br>[Shared Framework][OpenSUSE-42-sharedfx-6.0.X-coreclr] ([Checksum][OpenSUSE-42-sharedfx-checksum-6.0.X-coreclr]) |
+| **Linux-musl (x64)** | [![][linux-musl-x64-badge-6.0.X-coreclr]][linux-musl-x64-version-6.0.X-coreclr]<br>[tar.gz][linux-musl-x64-targz-6.0.X-coreclr] ([Checksum][linux-musl-x64-targz-checksum-6.0.X-coreclr])<br>[NetHost (tar.gz)][linux-musl-x64-nethost-targz-6.0.X-coreclr]<br>[Symbols (tar.gz)][linux-musl-x64-symbols-targz-6.0.X-coreclr] |
+| **Linux-musl (arm)** | [![][linux-musl-arm-badge-6.0.X-coreclr]][linux-musl-arm-version-6.0.X-coreclr]<br>[tar.gz][linux-musl-arm-targz-6.0.X-coreclr] ([Checksum][linux-musl-arm-targz-checksum-6.0.X-coreclr])<br>[NetHost (tar.gz)][linux-musl-arm-nethost-targz-6.0.X-coreclr]<br>[Symbols (tar.gz)][linux-musl-arm-symbols-targz-6.0.X-coreclr] |
+| **Linux-musl (arm64)** | [![][linux-musl-arm64-badge-6.0.X-coreclr]][linux-musl-arm64-version-6.0.X-coreclr]<br>[tar.gz][linux-musl-arm64-targz-6.0.X-coreclr] ([Checksum][linux-musl-arm64-targz-checksum-6.0.X-coreclr])<br>[NetHost (tar.gz)][linux-musl-arm64-nethost-targz-6.0.X-coreclr]<br>[Symbols (tar.gz)][linux-musl-arm64-symbols-targz-6.0.X-coreclr] |
+
+<!-- END generated table -->
+
+*Note: Our Linux packages (.deb and .rpm) are put together slightly differently than the Windows and Mac specific installers. Instead of combining everything, we have separate component packages that depend on each other. If you're installing these directly from the installer files (via dpkg or similar), then you'll need to install them in the order presented above.*
+
 ## More Advanced Scenario - Using your local CoreFx build
```

Comment:
This whole section should be deleted. The `Microsoft.Private.CoreFx.NETCoreApp` package that it talks about does not exist anymore.